com.unity.render-pipelines.core/Editor/Lighting/GUIFramework/ClickAction.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using UnityEngine;
+
+namespace UnityEditor
+{
+    /// <summary>
+    /// Action for click
+    /// </summary>
+    public class ClickAction : HoveredControlAction
+    {
+        private int m_Button;
+        private bool m_UseEvent;
+        /// <summary>
+        /// Click count
+        /// </summary>
+        public int clickCount = 1;
+        /// <summary>
+        /// Action during onClick
```

Comment:
The [Action](https://docs.microsoft.com/en-us/dotnet/api/system.action-1) to execute when the user satisfies the trigger condition.